pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -0,0 +1,171 @@
+--role-binding: The RoleBinding for the Service Account. It will bind the Service Account to the member cluster role
+`, "\n")
+
+func (o *accessTokenOptions) validateAndComplete() error {
+	if o.namespace == "" {
+		return fmt.Errorf("the namespace cannot be empty")
+	}
+	if o.serviceAccount == "" {
+		return fmt.Errorf("the service-account cannot be empty")
+	}
+	if o.roleBinding == "" {
+		return fmt.Errorf("the role-binding cannot be empty")
+	}
+	return nil
+}
+
+func NewAccessTokenCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "access-token",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Create a access-token in a Kubernetes cluster",
+		Long:    "Create a access-token in a Kubernetes cluster",
+		Example: accessTokenExamples,
+		RunE:    accessTokenRunE,
+	}
+
+	o := &accessTokenOptions{}
+	accessTokenOpts = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of the ClusterClaim")
+	command.Flags().StringVarP(&o.serviceAccount, "service-account", "", "", "Service Account of the access token")
+	command.Flags().StringVarP(&o.roleBinding, "role-binding", "", "", "RoleBinding of the Service Account")
+
+	return command
+}
+
+func accessTokenRunE(cmd *cobra.Command, args []string) error {
+	if err := accessTokenOpts.validateAndComplete(); err != nil {
+		return err
+	}
+	if len(args) != 1 {
+		return fmt.Errorf("exactly one NAME is required, got %d", len(args))
+	}
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: multiclusterscheme.Scheme})
+	if err != nil {
+		return err
+	}
+
+	serviceAccount := corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      accessTokenOpts.serviceAccount,
+			Namespace: accessTokenOpts.namespace,
+		},
+	}
+
+	fmt.Fprintf(cmd.OutOrStdout(), "creating ServiceAccount %s \n", accessTokenOpts.serviceAccount)
+	if err := k8sClient.Create(context.TODO(), &serviceAccount); err != nil {
+		if errors.IsAlreadyExists(err) {
+			fmt.Fprintf(cmd.OutOrStderr(), fmt.Sprintf("the ServiceAccount %s is already exists\n", accessTokenOpts.serviceAccount))
+		} else {
+			return err
+		}
+	}
+
+	fmt.Fprintf(cmd.OutOrStdout(), "creating RoleBinding %s \n", accessTokenOpts.roleBinding)
+	roleBinding := rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      accessTokenOpts.roleBinding,
+			Namespace: accessTokenOpts.namespace,
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "Role",
+			Name:     "antrea-mc-member-cluster-role",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      accessTokenOpts.serviceAccount,
+				Namespace: accessTokenOpts.namespace,
+			},
+		},
+	}
+	fmt.Fprintf(cmd.OutOrStdout(), "creating RoleBinding %s \n", accessTokenOpts.serviceAccount)
```

Comment:
you have two duplicate lines reporting `creating RoleBinding` and the parameter is different, maybe you can remove this line since `accessTokenOpts.serviceAccount` is incorrect.

Reply:
done